Base/QTApp/Resources/UI/qSlicerErrorReportDialog.ui

```diff
@@ -80,7 +80,17 @@
      <item>
       <widget class="ctkPushButton" name="LogFileOpenPushButton">
        <property name="text">
-        <string>Open log file in editor</string>
+        <string>Open log in editor</string>
+       </property>
+       <property name="autoDefault">
+        <bool>false</bool>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="ctkPushButton" name="LogOpenFileLocationPushButton">
+       <property name="text">
+        <string>Open log file location</string>
```

Comment:
Since in the other buttons you have dropped "file" such as `Open log file in editor` -> `Open log in editor`, then maybe best to keep the same convention here so:
```suggestion
        <string>Open log location</string>
```

Reply:
Updated